tests/cops_ui/test_cops_e2e.py

```diff
@@ -0,0 +1,117 @@
+import requests
+from tests import markers
+from selenium.webdriver.common.keys import Keys
+
+import pytest
+
+import json
+import urllib
+from urllib.request import urlopen
+
+from urllib.error import HTTPError, URLError
+from requests.exceptions import ConnectionError
+
+from time import sleep
+
+
+"""
+End to end test of cops(-staging).openstax.org
+Creates jobs for 2 collections, verifies that they were successfully executed and pdf is created
+Latest update on 26/02/2020
+"""
+
+
+@markers.parametrize(
+    "colid, style, bserver", [("col11992", "astronomy", "qa"), ("col11496", "anatomy", "staging")]
+)
+def test_cops_ui(selenium, cops_base_url, create_new_pdf_job, create_button, colid, style, bserver):
+
+    # GIVEN a cops base URL
+    # WHEN making a request to cops
+
+    try:
+
+        # verifying that cops URL is active
+        response = requests.get(cops_base_url)
+
+    except HTTPError:
+        pytest.exit("<<<<<<<<<< HTTP Error: COPS URL IS INVALID OR UNREACHABLE >>>>>>>>>>")
+    except URLError:
+        pytest.exit("<<<<<<<<<< URL Error: COPS URL IS INVALID OR UNREACHABLE >>>>>>>>>>")
+    except ConnectionError:
+        pytest.exit("<<<<<<<<<< Connection Error: COPS URL IS INVALID OR UNREACHABLE >>>>>>>>>>")
+
+    else:
+
+        # opening 'CREATE A NEW PDF JOB' dialog
+        selenium.get(cops_base_url)
```

Comment:
Is there a reason we aren't using a page object here?

Reply:
Remember the quote Otto,

> If you have WebDriver APIs in your test methods, You're Doing It Wrong. Simon Stewart

Reply:
I am not page object but I can investigate.